hyundai_kia_connect_api/KiaUvoApiCA.py

```diff
@@ -424,6 +424,7 @@ def _get_pin_token(self, token: Token, vehicle: Vehicle) -> None:
         )
         _LOGGER.debug(f"{DOMAIN} - Received Pin validation response {response.json()}")
         result = response.json()["result"]
+        token.last_action_pin_token = result["pAuth"]
```

Comment:
I would rename this to last_action_pauth.  It isn't actually a full token but rather just a pin authorization.

Reply:
I agree. I followed the naming convention of the existing method _get_pin_token, maybe we rename _get_pin_token to _get_pauth as well?

Reply:
Good call!

Reply:
finally got this done, last_pauth and last_action_id stored per vehicle